tests/fast-integration/test/fixtures/commerce-mock/index.js

```diff
@@ -919,10 +961,15 @@ async function getVirtualServiceHost(targetNamespace, funcName) {
   return vs.spec.hosts[0];
 }
 
-async function checkInClusterEventDeliveryHelper(targetNamespace, encoding, testSubscriptionV1Alpha2=false) {
+async function checkInClusterEventDeliveryHelper(targetNamespace, encoding, testSubscriptionV1Alpha2=false,
+    eventType='', eventSource='') {
   const eventId = getRandomEventId(encoding);
-  const eventType = testSubscriptionV1Alpha2? eventTypeOrderCompleted: '';
-  const eventSource = testSubscriptionV1Alpha2? eventSourceInApp: '';
+  if (!eventType) {
+    eventType = testSubscriptionV1Alpha2 ? eventTypeOrderCompleted : '';
+  }
+  if (!eventSource) {
+    eventSource = testSubscriptionV1Alpha2 ? eventSourceInApp : '';
+  }
```

Comment:
We can also move these if blocks to `checkInClusterEventDelivery(targetNamespace, testSubscriptionV1Alpha2=false)` so everything related to eventtypes are in one place for `v1alpha2`.

Reply:
Unfortunately, it is handled differently for legacy events. Otherwise, yes it'd be good idea to move it. Moreover, I haven't introduced it here.